src/io/flutter/run/daemon/DaemonApi.java

```diff
@@ -217,7 +217,7 @@ private Command takePending(int id) {
    * Parse the given string; if it is valid JSON - and a valid Daemon message - then return
    * the parsed JsonObject.
    */
-  static JsonObject parseAndValidateDaemonEvent(String message) {
+  public static JsonObject parseAndValidateDaemonEvent(String message) {
```

Comment:
it would be nice to keep this private. see comment above about only having to call it once per message.

Reply:
👍